src/Entity/Project/Cca.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+namespace App\Entity\Project;
+
+use App\Entity\Personne\Prospect;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Table()
+ * @ORM\Entity(repositoryClass="App\Repository\Project\CcaRepository")
+ * @UniqueEntity("nom")
+ */
+class Cca extends DocType
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="date")
+     */
+    private $dateFin;
+
+    /**
+     * @ORM\OneToMany(targetEntity=Etude::class, mappedBy="cca")
+     */
+    private $etudes;
+
+    /**
+     * @Assert\Valid()
+     * @ORM\ManyToOne(targetEntity=Prospect::class, cascade={"persist"})
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $prospect;
+
+    /************************
+     *   OTHERS DEFINITIONS
+     ************************/
+
+    /**
+     * @var bool
+     */
+    private $knownProspect = false;
+
+    /**
+     * @var Prospect
+     */
+    private $newProspect;
+
+    /**
+     * @ORM\Column(type="string", length=50, nullable=false, unique=true)
+     */
+    private $nom;
+
+    /**
+     * @ORM\OneToMany(targetEntity=Ce::class, mappedBy="cca")
+     */
+    private $bdcs;
+
+    public function __toString()
+    {
+        return $this->getNom() . ' - ' . $this->dateFin->format('d/m/Y');
```

Comment:
Inclure un identifiant de document pour supprimer toute ambiguité.

```suggestion
        return 'CCA '.$this->getNom() . ' - ' . $this->dateFin->format('d/m/Y');
```

Reply:
Yep, je pense que ça va poser un petit défaut d’affichage dans une prochaine PR mais je m’en occuperai